packages/fxa-settings/src/components/InputCheckboxBlue/index.tsx

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ChangeEvent, useState, useCallback } from 'react';
+import classNames from 'classnames';
+import { ReactComponent as CheckWhite } from './icon-check-white.svg';
+
+export type InputCheckboxBlueProps = {
+  defaultChecked?: boolean;
+  disabled?: boolean;
+  label?: string;
+  onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
+  prefixDataTestId?: string;
+};
+
+// This InputCheckboxBlue component is modeled on the most recently updated styles for non-authenticated flows
+export const InputCheckboxBlue = ({
+  defaultChecked,
+  disabled,
+  label,
+  onChange,
+  prefixDataTestId,
+}: InputCheckboxBlueProps) => {
+  const [checked, setChecked] = useState<boolean>(
+    defaultChecked ? true : false
```

Comment:
If `defaultChecked` is already a Boolean, you can skip the extra `? true : false` check.
Otherwise you might be able to do something like `!!defaultChecked` to cast to a Boolean.